src/test/java/com/sonyericsson/jenkins/plugins/bfa/model/FailureCauseProjectActionHudsonTest.java

```diff
@@ -43,16 +46,22 @@ public class FailureCauseProjectActionHudsonTest extends HudsonTestCase {
      */
     @Test
     public void testShowLastFailureOnProjectPage() throws Exception {
-        assumeFalse(Functions.isWindows());
```

Comment:
Now that we are running the test suite on Windows, fix this test so that it works on either Windows or Linux.